releasenotes/notes/plot-hop-vs-ntrials-aa4b8d80808edfdb.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Add :meth:`qiskit.ignis.verification.quantum_volume.QVFitter.plot_hop_accumulative` for plotting heavy output probability (HOP) vs number of trails similar to Figure 2a of QV64 paper (arXiv:2008.08571).
```

Comment:
Looks good - only a misprint:
trails --> trials

Reply:
Thank you :) I fixed the typos. Otherwise the code is good?